R/ci.R

```diff
@@ -54,6 +54,9 @@ CI <- R6Class(
     get_commit = function() {
       stop("NYI")
     },
+    can_push = function() {
+      Sys.getenv("id_rsa") != ""
```

Comment:
The base class is supposed to throw in all functions. The implementation would be a constant `TRUE` for local runs.